cashews/utils/bitarray.py

```diff
@@ -0,0 +1,29 @@
+class Bitarray:
+    __slots__ = ("_value",)
+
+    def __init__(self, value, base=10):
+        self._value = int(value, base)
+
+    def get(self, index: int, size: int = 1) -> int:
```

Comment:
Seems like this method can re-written using bit-wise operator. 

Basically this method will do almost the same:
```python
def get(value, idx, size):
    return value << size & (idx + 1)
```